lib/releases.js

```diff
@@ -144,11 +144,15 @@ const generateChangeLog = (mergedPullRequests, config) => {
       acc.push(
         categorizedPullRequests[category]
           .map(pullRequest =>
-            template(config['change-template'], {
-              $TITLE: pullRequest.title,
-              $NUMBER: pullRequest.number,
-              $AUTHOR: pullRequest.user.login
-            })
+            template(
+              config['change-template'],
+              {
+                $TITLE: pullRequest.title,
+                $NUMBER: pullRequest.number,
+                $AUTHOR: pullRequest.user.login
+              },
+              config['replacers']
```

Comment:
Minor nitpick, but could we change this to:
```suggestion
              config.replacers
```

Reply:
I would have totally thought eslint would have caught this. I kinda just copy and replaced it